src/Coordination/KeeperSnapshotManager.cpp

```diff
@@ -124,6 +126,7 @@ namespace
         readBinary(node.stat.numChildren, in);
         readBinary(node.stat.pzxid, in);
         readBinary(node.seq_num, in);
+        readBinary(node.size_bytes, in);
```

Comment:
Should we change changelog category to Backward Incompatible Change?

Reply:
> Should we change changelog category to Backward Incompatible Change?

I think there is no need to do compatible change, because the clickhouse-keeper is not release in production now, it is ongoing development state now.

Reply:
No, we should just update snapshot version and check it on snapshot read.

Reply:
Done.